tests/test_fits.py

```diff
@@ -128,6 +128,22 @@ def testUndefinedVector(self):
         metadata = self.writeAndRead(header)
         self.assertEqual(metadata.getArray("FOO"), [None, None])
 
+    def testNamedHeaderNavigate(self):
+        testfile = os.path.join(testPath, "data", "multi_extension_metadata.fits")
+
+        # load metadata from the extra table header
+        md_extra_tab = lsst.afw.fits.readMetadata(testfile, hduName="EXTRA_TAB")
+        # assert the value we put in is in the read metadata
+        self.assertTrue("FVALUE" in md_extra_tab)
+
+        # load metadata from the extra image header, do same test
+        md_extra_im = lsst.afw.fits.readMetadata(testfile, hduName="EXTRA_IM")
+        self.assertTrue("BLORP" in md_extra_im)
+
+        # now try to load a non-existent named HDU and check that we throw
+        with self.assertRaises(lsst.afw.fits.FitsError):
+            lsst.afw.fits.readMetadata(testfile, hduName="CORDON_BLEAU")
```

Comment:
I guess `cordon bleu` isn't found either ;-)

Reply:
quite so ;-). Again, I didn't see any conventions mandated for test values but can change them to whatever.